### ISSUE TITLE
kraken2 does not generate any result due to wrong PATH to its database in config_file.txt

### ISSUE THREAD

**Issue by nmquijada** [nmquijada/tormes#30]
There is a bug in ```tormes-setup``` while generating the PATH for the kraken2 database in the *config_file.txt*.
If you are not using your own kraken2 database, just the one installed by default in TORMES, please type the following commands to solve this issue:

```
conda activate tormes-1.2.0 #skip in case the environment is already loaded
sed -i "s/minikraken_8GB_202003$/minikraken_8GB_20200312/" $(which tormes | sed "s/bin\/tormes/files\/config_file.txt/")
```

This issue will be solved in the next TORMES release.
Sorry for the inconveniences.
Narciso

**Comment by nmquijada** [comment]
Fixed in ```version 1.2.1```

**Comment by Panchastico** [comment]
This may work for version 1.3.0 as well. I have the same problema in version 1.3.0
